c++/42.Trapping Rain Water/42.Trapping Rain Water.cpp

```diff
@@ -0,0 +1,22 @@
+class Solution {
```

Comment:
First line should be Question link!!

Reply:
I have made the above change can you please check